app/services/katello/organization_creator.rb

```diff
@@ -45,9 +46,6 @@ def create!
         seed!
 
         create_backend_objects!
-
-        @organization.created_in_katello = true
```

Comment:
I think `created_in_katello` is meant to track whether the Candlepin owner and library environment are correctly in place. Setting `created_in_katello` to true before running `create_backend_objects` will defeat the purpose, no?  Let's ask the original author @jturel for more comment.  Sorry I am being so particular but I wouldn't want to re-break this especially without testing it :wink:

Reply:
Sure, no worries :+1:  If you have concerns, let's solve those first :+1:

Reply:
It all happens in a transaction so there should be no functional difference - no test failure leads me to believe that especially. I probably put the assignment to created_in_katello to true where I did per Jeremy's comment above - only setting true after the Candlepin and other backend parts have happened - so it's more of a conceptual thing.